test/integration/scan_test.go

```diff
@@ -371,7 +371,6 @@ func TestContainerEngineScansE2E_ContainerImagesFlagAndScanType(t *testing.T) {
 		defer deleteProject(t, projectID)
 		assert.Assert(t, scanID != "", "Scan ID should not be empty")
 		assert.Assert(t, projectID != "", "Project ID should not be empty")
-		assertZipFileRemoved(t)
```

Comment:
why you removed it?

Reply:
@BenAlvo1  Claims that there is a problem with it that it causes the tests to fall
Because sometimes the test fails to delete immediately
Then the next test falls right after it

Reply:
@AlvoBen